core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/GenerateDot.scala

```diff
@@ -88,7 +88,7 @@ object GenerateDot {
     val accumSummary = accums.map { a =>
       Seq(a.sqlID, a.accumulatorId, a.total)
     }
-    val accumIdToStageId = app.accumIdToStageId
+    val accumIdToStageId = app.stageManager.reduceAccumMapping()
```

Comment:
This is a hack to get the generateDot to work with the 1-to-M map.
`GenerateDot` is rarely used and fixing the implementation to be 1-to-M is going to bloat the PR